dpti/hti_liq.py

```
@@ -28,45 +28,64 @@ def make_iter_name(iter_index):
+    if epsilon is not None:
+        assert (
+            epsilon_0_0 is None
+        ), "epsilon and epsilon_0_0 cannot be set at the same time"
+        ret += f"variable        EPSILON equal {epsilon:f}\n"
+        for i, j in sigma_key_index:
+            ret += "pair_coeff      {} {} {p:s}${{EPSILON}} {:f} {:f}\n".format(
+                i + 1,
+                j + 1,
+                pair_coeff_str,
+                sparam["sigma_" + str(i) + "_" + str(j)],
+            )
```

Ruff (0.8.2)

`.format` call is missing argument(s) for placeholder(s): p

(F524)